frontend/src/App.js

```diff
@@ -33,19 +40,26 @@ export default function App() {
             <Redirect to={`/details/${singleTrack.id}`} />
           ) : (
             <>
-              <Header goBackFunction={setStartingPoint} />
+              <Header goBackFunction={setStartingPoint} redirectToPath="/" />
               <Results
                 startingPoint={startingPoint}
                 setSingleTrack={setSingleTrack}
+                bookmarks={bookmarks}
+                setBookmarks={setBookmarks}
               />
```

Comment:
Could be setted up in a page and then imported-> just a structure hint

Reply:
Good point, I'll think about that